Controllers/StaffController.cs

```diff
@@ -20,17 +20,9 @@ public StaffController(stunning_robot_HRContext context)
         }
 
         // GET: Staff
-        public async Task<IActionResult> Index(string searchString)
+        public async Task<IActionResult> Index()
         {
-            var staff = from m in _context.Staff
-                select m;
-
-            if (!String.IsNullOrEmpty(searchString))
-            {
-                staff = staff.Where(s => s.LastName.Contains(searchString));
-            }
-
-            return View(await staff.ToListAsync());
+            return View(await _context.Staff.ToListAsync());
```

Comment:
I think you lost some functionality here Erin. The method was doing what you wanted above, but optionally allowing you to provide a string to search the staff by last name. Did you remove purposely?